src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java

```diff
@@ -174,10 +185,10 @@ public Series getSingleSeries(QueryBuilder queryBuilder) throws DicomFuseExcepti
         .addParameter(PARAM_INCLUDE_FIELD, VALUE_PARAM_STUDY_INSTANCE_UID)
         .addParameter(PARAM_SERIES_ID, queryBuilder.getSeriesId())
         .setPath(path);
-    List<Series> series = createRequestForObjectList(uriBuilder, new TypeReference<List<Series>>() {
-    });
-    if (series.size() < 1) {
-      throw new DicomFuseException("Series not found");
+    List<Series> series =
+        createRequestForObjectList(uriBuilder, new TypeReference<List<Series>>() {});
+    if (series.size() == 0) {
+      throw new DicomFuseException("Series not found!");
```

Comment:
Let's avoid adding exclamation marks to error messages. It's already an error, no need to emphasize more.

Reply:
Ok, I understood. Done